src/w3c/headers.js

```diff
@@ -395,11 +395,22 @@ export async function run(conf) {
   }
 
   if (conf.isEd) conf.thisVersion = conf.edDraftURI;
-  if (conf.isCGBG) validateCGBG(conf);
+  if (conf.isCGBG) {
+    validateCGBG(conf);
+  }
+  if (conf.isTagEditorFinding && !conf.latestVersion) {
+    conf.latestVersion = null;
+  }
   if (conf.latestVersion !== null) {
     conf.latestVersion = conf.latestVersion
       ? w3Url(conf.latestVersion)
       : w3Url(`${pubSpace}/${conf.shortName}/`);
+    const exists = await resourceExists(conf.latestVersion);
```

Comment:
I'm concerned this can cause issue during publish (via spec-prod).

Reply:
Can you help me understand what the issue might be? Is it the warning?

Reply:
Sorry, I misunderstood it as `thisVersion`.